docs/1.getting-started/10.deployment.md

```diff
@@ -81,52 +81,61 @@ There are two ways to deploy a Nuxt application to any static hosting services:
 
 ### Crawl-based Pre-rendering
 
-Use the [`nuxi generate` command](/docs/api/commands/generate) to build your application. For every page, Nuxt uses a crawler to generate a corresponding HTML and payload files. The built files will be generated in the `.output/public` directory.
+Use the [`nuxi generate` command](/docs/api/commands/generate) to build and pre-render your application using the [Nitro](/docs/guide/concepts/server-engine) crawler. This command is similar to `nuxt build` with the `nitro.prerender` option set to `true`.
```

Comment:
```suggestion
Use the [`nuxi generate` command](/docs/api/commands/generate) to build and pre-render your application using the [Nitro](/docs/guide/concepts/server-engine) crawler. This command is similar to `nuxt build` with the `nitro.static` option set to `true`, or running `nuxt build --prerender`.
```

Reply:
🤔 There is no such option in the Nitro API https://nitro.unjs.io/config#static.

Will investigate in the Nitro documentation.

Reply:
Ok, I see, that's map to a preset.

https://github.com/unjs/nitro/blob/main/src/presets/static.ts